### PR TITLE
Add power-meter asset subtype

### PR DESCRIPTION
**Pull request by aquette**
Signed-off-by: Arnaud Quette <ArnaudQuette@Eaton.com>

### CODE REVIEW

**Comment by aquette** [comment]
No error with me ;)
16 comes from emc4j...

File: database/mysql/Makefile.am [hunk 1]
```diff
@@ -17,7 +17,8 @@ mysql_DATA	= initdb.sql \
 			0012_device_type_extension.sql \
 			0013_user_management.sql \
 			0014_user_preferences.sql \
-			0015_ipm2.sql 
+			0015_ipm2.sql  \
+			0017_device_type_extension.sql
```
[the remark this answers is not in the document]
Comment (jimklimov) [reply database/mysql/Makefile.am]: The joys of mixing two worlds... the 016 is in automation repos (and image builds).

**Comment by jimklimov** [comment]
Looks sane, can merge when the feature comes to fruition overall.

**Comment by aquette** [comment]
Closing in favor of https://github.com/42ity/fty-core/pull/353

**Pull request closed without merging**